pkg/client/public/locales/es/translation_old.json

```diff
@@ -0,0 +1,16 @@
+{
+    "terms": {
+        "identity": "",
+        "createdBy": "Creado Por",
+        "type": "Tipo"
+    },
+    "dialog": {
+        "title": {
+            "newIdentity": "",
+            "updateIdentity": ""
+        }
+    },
+    "message": {
+        "selectIdentityType": ""
+    }
+}
```

Comment:
`translation_old.json` files don't have to be checked in anymore as they have been added to `.gitignore`